### PR TITLE
portable python

### PR DESCRIPTION
**Pull request by scopatz**
CC @kcpevey

### CODE REVIEW

File: posts/2020/06/portable-python-env.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+Cleanup prefixes from inside the active environment.
+Note that this command can also be run without activating the environment
+as long as some version of python is already installed on the machine.
+
+`(my_env) $ conda-unpack`
+
+At this point the environment is exactly as if you installed it here
+using conda directly. All scripts should be fully functional, e.g.:
+
+`(my_env) $ ipython --version`
```
Comment (mattip) [inline posts/2020/06/portable-python-env.md:159]: Don’t know if it matters for this example, but ipython wasn’t one of the explicit dependencies. Will it be installed?

Comment (scopatz) [reply posts/2020/06/portable-python-env.md:161]: Good catch! I added it

**Comment by kcpevey** [comment]
Thanks @scopatz and @mattip !

**Comment by rgommers** [comment]
Nice post @kcpevey. And thanks @scopatz for the PR and @mattip for reviewing. Merged, should be up in a minute.

**Pull request merged**